Paper.tex

```diff
@@ -319,7 +319,7 @@ \subsection{The Transaction} \label{ch:transaction}
 \mathbb{B}_{0} & \text{otherwise}\end{cases}
 \end{equation}
 
-\subsection{The Block} \label{ch:block}
```

Comment:
This label is used twice, so `\ref{ch:block}` occurrences need to be replaced as well.